src/se/content.rs

```diff
@@ -0,0 +1,614 @@
+//! Contains serializer for content of an XML element
+
+use crate::errors::serialize::DeError;
+use crate::se::element::{ElementSerializer, Struct};
+use crate::se::simple_type::{QuoteTarget, SimpleTypeSerializer};
+use crate::se::{QuoteLevel, XmlName};
+use serde::ser::{
+    Impossible, Serialize, SerializeSeq, SerializeTuple, SerializeTupleStruct, Serializer,
+};
+use serde::serde_if_integer128;
+use std::fmt::Write;
+
+macro_rules! write_primitive {
+    ($method:ident ( $ty:ty )) => {
+        #[inline]
+        fn $method(self, value: $ty) -> Result<Self::Ok, Self::Error> {
+            self.into_simple_type_serializer().$method(value)
+        }
+    };
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// A serializer used to serialize content of the element. It does not write
+/// surrounding tags.
+///
+/// This serializer does the following:
+/// - primitives (booleans, numbers and strings) serialized as naked strings
+/// - `None` does not write anything
+/// - sequences serialized without delimiters
```

Comment:
An example would be nice, similar to the others

Reply:
Added an example: `[1, 2, 3]` would be serialized as `123`